lib/translator/language_based_translator.rb

```diff
@@ -0,0 +1,56 @@
+require 'yaml'
+require_relative 'google_translator_api'
+
+module Translator
+  class LanguageBasedTranslator
+    CONFIG = {
+      classes: ["Array", "Class", "Object", "Integer", "Math"],
+      methods: {
+        public_methods: 'cpumethods',
+        private_methods: 'cprmethods',
+        instance_methods: 'ipumethods',
+        private_instance_methods: 'iprmethods'
+      }
+    }.freeze
```

Comment:
Is there any significance of cpumethods, cprmethods. ipumethods and iprmethods ?